cve_bin_tool/checkers/subversion.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/python3
+
+"""
+CVE checker for subversion CLI
+References:
+https://subversion.apache.org/
+https://www.cvedetails.com/vulnerability-list/vendor_id-2120/product_id-3608/Subversion-Subversion.html
+"""
+from . import Checker
+
+
+class SubversionChecker(Checker):
+
+    CONTAINS_PATTERNS = [
+        r"Working copy locked; if no other Subversion client is currently using the working copy, try running 'svn cleanup' without the --remove",
+        r"Working copy locked; try running 'svn cleanup' on the root of the working copy ('%s') instead.",
+    ]
+    FILENAME_PATTERNS = [
+        r"subversion",
+        r"svn",
+        r"libapache2-svn",
+        r"libsvn-dev",
+        r"svnadmin",
+        r"svnauthz",
+        r"svnauthz-validate",
+        r"svnbench",
+        r"svndumpfilter",
+        r"svnfsfs",
+        r"svnlook",
+        r"svnmucc",
+        r"svnrdump",
+        r"svnserve",
+        r"svnsync",
+        r"svnversion",
+    ]
+    VERSION_PATTERNS = [
+        r"subversion-([0-9]+\.[0-9]+\.[0-9]+)",
+        r"Cannot guarantee compatibility beyond the current running version (%s)\n([0-9]+\.[0-9]+\.[0-9]+)\nhelp",
```

Comment:
This doesn't seem very good signature since it isn't mentioning SVN or subversion anywhere. If you want to keep it you need to make sure the filename satisfy filename_patterns. You can do this by overloading default get_version method. We have done that in Python and SQLite.